pkg/server/server.go

```diff
@@ -3,8 +3,13 @@ package server
 import (
 	"crypto/ecdsa"
 	"crypto/x509"
+	"github.com/spiffe/spire/pkg/common/profiling"
```

Comment:
nit: this should be further down w/ the rest of the github imports

Reply:
:100: